content/docking-station.md

```diff
@@ -59,6 +60,9 @@ Community members have reported that the following docks work with our products:
 - [Anker A8392](https://us.anker.com/products/a8392) [[community-tested](https://github.com/system76/docs/pull/797) on an Intel system] <sup>1</sup>
 - [CalDigit TS3 Plus](https://www.caldigit.com/ts3-plus/) [[community-tested](https://github.com/system76/docs/pull/417) on an Intel system] <sup>1</sup>
   - Downstream (passthrough) Thunderbolt 3 port not tested.
+  - May not be able to wake system using peripherals connected via dock.
+  - These issues can be potentially addressed by changing the Thunderbolt Security Mode to "None" in the BIOS, though this is not recommended.
+    - Displays sometimes don't wake up from sleep until dock is re-plugged.
```

Comment:
I'm a little confused by the indentation here-- is it the displays issue that can be resolved by changing the security level? If so, I would swap the indentations so the workaround is a sub-bullet of the issue.